tests/virtctl/usbredir.go

```diff
@@ -187,20 +184,69 @@ var _ = Describe("[crit:medium][vendor:cnv-qe@redhat.com][level:component][sig-c
 })
 
 func runConnectGoroutine(
-	virtClient kubecli.KubevirtClient,
 	name string,
 	namespace string,
 	ctx context.Context,
 	errch chan error,
 ) {
-	defer GinkgoRecover()
+	cmd := newVirtctlCommand("usbredir",
+		"--namespace", namespace,
+		"--no-launch",
+		name,
+	)
+	// To find ip/port to connect
+	rOut, wOut := io.Pipe()
+	cmd.SetOut(wOut)
+
+	// To find errors
+	rErr, wErr := io.Pipe()
+	cmd.SetErr(wErr)
+
+	remote := make(chan error)
+	go func() {
```

Comment:
Are you sure all of these routines exit cleanly, i.e. without leaking?

Reply:
Probably the test-for-flakes has spotted something here indeed, I"ll check.